### PR TITLE
Update CHANGELOG per PR 2162.

### PR DESCRIPTION
**Pull request by momuno**

### CODE REVIEW

**Review by ahsonkhan — APPROVED** [review]
Looks good. Thanks!

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 ### Bugs Fixed
 
 - [[#2152]](https://github.com/Azure/azure-sdk-for-c/pull/2152) Fix value in user agent string.
+- [[#2162]](https://github.com/Azure/azure-sdk-for-c/pull/2162) Remove $version requirement in reported properties response topic.
```
Comment (ahsonkhan) [inline CHANGELOG.md:8]:
```suggestion
- [[#2162]](https://github.com/Azure/azure-sdk-for-c/pull/2162) Remove $version requirement in reported IoT Hub and Edge properties response topic.
```

**Comment by momuno** [comment]
@danewalton @CIPop @ewertons thoughts on the wordsmithing of the tagline?

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,7 @@
 ### Bugs Fixed
 
 - [[#2152]](https://github.com/Azure/azure-sdk-for-c/pull/2152) Fix value in user agent string.
+- [[#2162]](https://github.com/Azure/azure-sdk-for-c/pull/2162) Remove $version requirement in IoT Hub and IoT Edge reported properties response topic.
```
Comment (CIPop) [inline CHANGELOG.md:8]: One idea would be to add "Twin" reported properties to distinguish from telemetry ones.

**Review by CIPop — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (momuno) [reply CHANGELOG.md:8]: thanks for the feedback cris!

Comment (momuno) [reply CHANGELOG.md:8]: I don't want it to sound like $version will no longer be in the IoT Hub topic, and I'm concerned the current tag line could be interpreted to mean that...

Comment (momuno) [reply CHANGELOG.md]: Please see update.

**Comment by momuno** [comment]
Have to run, so merging PR.

**Review by danewalton — APPROVED** [review]

**Pull request merged**